Sample_v3/Samples/SimpleChat/SimpleChannelMembersViewController.swift

```diff
@@ -0,0 +1,201 @@
+    /// happen, `memberListController.delegate` must be equal `self` and `memberListController.synchronize()` must be called.
+    ///
+    
+    ///
+    /// # didChangeMembers
+    ///
+    /// The method below receives the `changes` that happen in the list of channel members and updates the `UITableView` accordingly.
+    ///
+    func memberListController(
+        _ controller: ChatChannelMemberListController,
+        didChangeMembers changes: [ListChange<ChatChannelMember>]
+    ) {
+        tableView.beginUpdates()
+        
+        for change in changes {
+            switch change {
+            case let .insert(_, index: index):
+                tableView.insertRows(at: [index], with: .automatic)
+            case let .move(_, fromIndex: fromIndex, toIndex: toIndex):
+                tableView.moveRow(at: fromIndex, to: toIndex)
+            case let .update(_, index: index):
+                tableView.reloadRows(at: [index], with: .automatic)
+            case let .remove(_, index: index):
+                tableView.deleteRows(at: [index], with: .automatic)
+            }
+        }
+        
+        tableView.endUpdates()
+    }
+    
+    // MARK: - UITableViewDataSource
+
+    ///
+    /// The methods below are part of the `UITableViewDataSource` protocol and will be called when the `UITableView`
+    /// needs information which will be given by the `memberListController` object.
+    ///
+    
+    ///
+    /// # numberOfRowsInSection
+    ///
+    /// The method below returns the current loaded users count `memberListController.members.count`.
+    /// It will increase as more users are loaded or decrease as users are deleted.
+    ///
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        memberListController.members.count
+    }
+
+    ///
+    /// # trailingSwipeActionsConfigurationForRowAt
+    ///
+    /// The method below returns a list of swipe actions available on the channel member.
+    ///
+    override func tableView(
+        _ tableView: UITableView,
+        trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath
+    ) -> UISwipeActionsConfiguration? {
+        let cid = memberListController.query.cid
+        let member = memberListController.members[indexPath.row]
+        
+        guard
+            let me = memberListController.members.first(where: { $0.id == memberListController.client.currentUserId }),
+            me.memberRole != .member,
+            me.id != member.id
+        else { return nil }
+
+        let removeAction = UIContextualAction(style: .destructive, title: "Remove") { [weak self] _, _, completion in
+            guard let self = self else { return }
+            
+            let channelController = self.memberListController.client.channelController(for: cid)
```

Comment:
nit: Can you create a local variable like this `let client = memberListController.client` in the scope of the function? It will make the code a tiny bit clearer.

Reply:
fixed in d8708db39a3c56bc1f670ad843c1a3991cf0ec48 and 4f7f9bee45bcaa346ad40a2a13beda87bcce5c36 for both `SimpleXXX` and `CombineSimpleXXX` controllers